routes.py

```diff
@@ -1,13 +1,36 @@
-from flask import Flask, request, Response, jsonify
+from flask import Flask, request, Response, jsonify, redirect, url_for, session
 from flask_cors import CORS, cross_origin
 from flask import Flask, make_response
 from scraper.scraper import get_profile, scrape
+from werkzeug.utils import secure_filename
 import json
+import os
+
 app = Flask(__name__)
 
+UPLOAD_FOLDER = '/Users/samurai/code/couleur/be_flask/'
```

Comment:
Use relative path here, so that when deployed on any machine, no rework is required.